source/MapPlanetCard.cpp

```diff
@@ -269,20 +294,17 @@ void MapPlanetCard::Highlight(double availableSpace) const
 
 double MapPlanetCard::AvailableTopSpace() const
 {
-	const Interface *planetCardInterface = GameData::Interfaces().Get("map planet card");
-	const double height = planetCardInterface->GetValue("height");
+	const double height = Height();
 	return min(height, max(0., (number + 1) * height - MapDetailPanel::GetScroll()));
 }
 
 
 
-double MapPlanetCard::AvailableBottomSpace() const
+double MapPlanetCard::AvailableBottomSpace()
```

Comment:
Since this function remains unchanged in the header, I assume it wasn't intentional.
```suggestion
double MapPlanetCard::AvailableBottomSpace() const
```